manage-apps/register.html

```diff
@@ -204,9 +206,15 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         success: function(data) {
-          console.log(data);
-          $("#sandboxRegistration").addClass('submited');
-          $('#form-success').removeClass('form-success-hidden');
+          if (data.sandboxErr !== 'No Error') {
```

Comment:
Now that I think about it, on the back end, I'm changing null to "No Error." I guess I could just keep it null, and check for that. What do you think?